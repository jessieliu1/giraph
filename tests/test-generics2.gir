int main() {
	wedigraph<string> g = [A:"hey" -{5}-> B:"you"];
	graph<string> g2 = [C:"its" -- D:"me"];
	for_edge(e : g2) {
		g.add_edge(e.from(), e.to(), 0);
		g.add_edge(e.to(), e.from(), 1);
	}
	g.set_edge_weight(C, D, g.get_edge_weight(C, D) * 100);
	g.set_edge_weight(D, C, g.get_edge_weight(D, C) * 100);
	for_node(n : g) {
		prints(n.data());
	}
	for_edge(e : g) {
		prints(e.from().data());
		prints(e.to().data());
		print(e.weight());
	}
	return 0;
}