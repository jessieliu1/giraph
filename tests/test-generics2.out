hey
you
its
me
hey
you
5
its
me
0
me
its
100
